libuuu/libuuu.h

```diff
@@ -108,8 +108,8 @@ struct uuu_notify
 
 typedef int (*uuu_notify_fun)(struct uuu_notify, void *data);
 
-int uuu_register_notify_callback(uuu_notify_fun f, void *data);
-int uuu_unregister_notify_callback(uuu_notify_fun f);
+bool uuu_register_notify_callback(uuu_notify_fun f, void *data);
+bool uuu_unregister_notify_callback(uuu_notify_fun f);
 
```

Comment:
Keep all return value as int. 0 - success. other value is error code

Reply:
Fixed :+1: